### ISSUE TITLE
[documentation] Give correct arguments to read simulators

### ISSUE THREAD

**Issue by yoann-dufresne** [CAMI-challenge/CAMISIM#53]
I'm trying to use CAMISIM to generate a big fastq file containing community reads with no error.

If I understand well, this is possible using wgsim read simulator with the option "errorfree" (I saw that in the code).
So, I changed the config file with theses values :
readsim=tools/wgsim/wgsim
error_profiles=errorfree

But this is not that simple.
CAMISIM wants a file as input for the error_profiles value.
How can I change the values for a working pipeline ?

I think that it should be nice to have one or more wiki pages to explain the possible values in the config file.

**Comment by AlphaSquad** [comment]
You are right, there should be a description for every read simulator how to modify the config accordingly, and the way to do this for the wgsim simulator is particularly ugly. I will need to have a look at that.

Having said that, the way it should work right now (changing the following 4 options):
readsim=tools/wgsim/wgsim
error_profiles=. 
\# CAMISIM expects a directory, but the option is not really required
profile=0.0 
\# error-rate in %
type=wgsim

**Comment by yazhinia** [comment]
How can we do the error-free or customized error rate read simulation with Illumina or other platform sequences?

**Comment by AlphaSquad** [comment]
Error-free can only be done with `wgsim` (see my comment above for this). It should not make a difference whether you use `ART` or `wgsim` here. 
For using the custom error profiles of `ART` please have a look at this pull request which describes how to add and use you own trained error profile: https://github.com/CAMI-challenge/CAMISIM/pull/88
